roles/mongodb/tasks/main.yml

```diff
@@ -0,0 +1,40 @@
+---
+- name: install mongodb packages
+  apt: pkg={{ item }} state=installed
+  with_items:
+    - mongodb-org-server={{ mongodb.version }}
+    - mongodb-org-shell={{ mongodb.version }}
```

Comment:
I just verified that manually upgrading mongo versions DOES trigger the mongod service to restart when upgrade is completed.

Therefore, I'm assuming if someone were to update the {{ mongodb.version }} after an initial install and rerun the deployment, then Ansible is smart enough to detect the version change and run that task, thus triggering mongodb to restart itself when complete.